Updates on 106-nodejs-command-line-apps (#6087)
kamranahmedse/developer-roadmap: src/data/roadmaps/nodejs/content/106-nodejs-command-line-apps/101-printing-output/102-chalk.md

# Chalk

> Chalk is a clean and focused library used to do string styling in your terminal applications. With it you can print different styled messages to your console like:
> - changing font colors
> - font boldness
> - font opacity 
> - also the background of any message printed on your console.
> - and more

Reviewer: ```suggestion
Chalk is a clean and focused library used to do string styling in your terminal applications. With it, you can print different styled messages to your console such as changing font colors, font boldness, font opacity, and the background of any message printed on your console.
```

Reviewer: We try to keep content as a single paragraph for consistency.
